app/query_objects/active_query.rb

```diff
@@ -0,0 +1,11 @@
+class ActiveQuery
+  delegate :all, :each, :for_each, to: :active
+
+  def initialize(model)
+    @relation = model.where(active: true)
+  end
+
+  def active
+    @relation
+  end
+end
```

Comment:
`Template.active` was a scope I had to test and maintain in `Template`. A handful of other models would were going to require the same scope. Also, I like to keep models as small and focussed as possible. `ActiveQuery` does one thing well, and if the definition of what active is should change in the future, we need only change the code in one place.